### PR TITLE
chore: remove unnecessary string hashes

### PR DESCRIPTION
**Pull request by hamirmahal**

### CODE REVIEW

**Review by hamirmahal — COMMENTED** [review]
This shouldn't result in any user-facing changes.

The goal of this PR is to improve code readability and maintainability.

File: src/aws/auth.rs [hunk 1]
```diff
@@ -388,8 +388,8 @@ mod tests {
     #[test]
     fn parsing_default() {
         let config = toml::from_str::<ComponentConfig>(
-            r#"
-        "#,
+            r"
+        ",
```
Comment (jorgehermo9) [inline src/aws/auth.rs:392]:
```suggestion
            "",
```

File: src/aws/auth.rs [hunk 2]
```diff
@@ -399,9 +399,9 @@ mod tests {
     #[test]
     fn parsing_default_with_load_timeout() {
         let config = toml::from_str::<ComponentConfig>(
-            r#"
+            r"
```
Comment (jorgehermo9) [inline src/aws/auth.rs:402]:
```suggestion
            "
```

File: src/aws/auth.rs [hunk 3]
```diff
@@ -435,11 +435,11 @@ mod tests {
     #[test]
     fn parsing_default_with_imds_client() {
         let config = toml::from_str::<ComponentConfig>(
-            r#"
+            r"
```
Comment (jorgehermo9) [inline src/aws/auth.rs:438]:
```suggestion
            "
```

File: src/aws/region.rs [hunk 4]
```diff
@@ -55,8 +55,8 @@ mod tests {
 
     #[test]
     fn optional() {
-        assert!(toml::from_str::<RegionOrEndpoint>(indoc! {r#"
-        "#})
+        assert!(toml::from_str::<RegionOrEndpoint>(indoc! {r"
```
Comment (jorgehermo9) [inline src/aws/region.rs:58]:
```suggestion
        assert!(toml::from_str::<RegionOrEndpoint>(indoc! {"
```

Comment (jorgehermo9) [reply src/aws/region.rs:58]: apply all of this in the rest of the changes please too please. It seems that the raw strings were also not necessary

**Comment by jszwedko** [comment]
Thanks @hamirmahal . I wonder if `clippy` will add a rule for this in the future. I agree with @jorgehermo9's comments, but otherwise this looks good.

**Comment by jszwedko** [comment]
(didn't mean to close)

**Comment by hamirmahal** [comment]
You're welcome @jszwedko.

**Comment by hamirmahal** [comment]
I'll see if I can cleanly batch together all of the suggestions, now.

File: src/transforms/aggregate.rs [hunk 5]
```diff
@@ -1004,9 +1004,9 @@ mod tests {
     #[tokio::test]
     async fn transform_shutdown() {
         let agg = toml::from_str::<AggregateConfig>(
-            r#"
+            r"
```
Comment (pront) [inline src/transforms/aggregate.rs:1007]: We can use `indoc!` here.

File: src/transforms/throttle.rs [hunk 6]
```diff
@@ -251,10 +251,10 @@ mod tests {
     async fn throttle_events() {
         let clock = clock::FakeRelativeClock::default();
         let config = toml::from_str::<ThrottleConfig>(
-            r#"
+            r"
```
Comment (pront) [inline src/transforms/throttle.rs:254]: We can use `indoc!` here and add indentation.

**Comment by hamirmahal** [comment]
You're welcome, @jorgehermo9!

I've got my hands full for the next couple of weeks so I don't think I'll be able to look into the remaining occurrences anytime soon.

What are your thoughts on accepting this pull request as-is, and then I or someone else can handle the remaining occurrences in a future PR?

**Comment by pront** [comment]
> What are your thoughts on accepting this pull request as-is, and then I or someone else can handle the remaining occurrences in a future PR?

Sounds reasonable 👍

**Review by jszwedko — APPROVED** [review]

**Pull request merged**